namespaces/live-1.cloud-platform.service.justice.gov.uk/kuberos/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: kuberos
+spec:
+  limits:
+  - default:
+      cpu: 250m
+      memory: 500Mi
+    defaultRequest:
```

Comment:
Kuberos pods in production use 1m of CPU. Is there a reason to set aside 125m per container?
Similarly, the running pods are using 16Mi of memory. Do we need to reserve 250Mi?